ivy_tests/test_ivy/test_functional/test_core/test_device.py

```diff
@@ -257,9 +255,8 @@ def test_to_device(
         assert ivy.dev(x_on_dev, as_native=True) == ivy.dev(out, as_native=True)
 
         # check if native arrays are the same
-        if ivy.current_backend_str() in ["tensorflow", "jax"]:
-            # these backends do not support native inplace updates
-            return
+        # these backends do not support native inplace updates
+        assume(not (ivy.current_backend_str() in ["tensorflow", "jax"]))
```

Comment:
let's use `fw` rather than `ivy.current_backend_str`, the same goes for all other tests.